common/api/ui-framework.api.md

```diff
@@ -1910,7 +1910,7 @@ export const expandWidget: (base: {
         readonly home: {
             readonly widgetIndex: number;
             readonly widgetId: string | undefined;
-            readonly side: PanelSide;
+            readonly side: "left" | "right" | "bottom" | "top";

```

Comment:
More extract-api noise...

Reply:
Seems more likely the presentation folks forgot to run extract-api. Addon PR will catch any problems.